Packs/CyberChef/Integrations/CyberChef/CyberChef.yml

```diff
@@ -3,16 +3,20 @@ commonfields:
   id: CyberChef
   version: -1
 configuration:
+- display: Use the local npm package to execute the bake function
```

Comment:
This parameter effects both `cyberchef-bake` and `cyberchef-magic` commands, so why its name only refers to the "bake" functionally? 
Could you also add a short `additionalinfo` to explain when one should use this parameter?